### PR TITLE
Update ts/README.md to include the default nav config

### PR DESCRIPTION
**Pull request by csharrison**

### CODE REVIEW

File: ts/README.md [hunk 1]
```diff
@@ -35,3 +35,8 @@ npm run flexible-event -- -w 1,2,3 -b 4,5,6
 ```sh
 npm run flexible-event -- -f /path/to/source_registration.json
 ```
+
+Here is an example which matches the default config for navigation sources:
+```sh
+npm run flexible-event -- -w 3,3,3,3,3,3,3,3 -b 3,3,3,3,3,3,3,3 -m 3 -t navigation
```
Comment (apasel422) [inline ts/README.md:41]: Maybe we can add a brief comment here about how these are unzipped tuples of a list of length `trigger data cardinality`.

**Review by apasel422 — APPROVED** [review]

**Comment by csharrison** [comment]
PTAQL

File: ts/README.md [hunk 2]
```diff
@@ -28,10 +28,23 @@ On responses:
...
 
+The `flexible-event` utility accepts comma-separated lists of the
+number of windows and summary buckets per trigger data. This is
+essentially an unzipped list of two-tuples, one per trigger data.
+
 ```sh
 npm run flexible-event -- -w 1,2,3 -b 4,5,6
 ```
 
+The utility also will parse (in a rudimentary way) the source registration
```
Comment (apasel422) [inline ts/README.md:39]:
```suggestion
The utility also will parse the source registration
```
It's not rudimentary anymore. I made it so it's fully parsed like the actual header validator.

Comment (csharrison) [reply ts/README.md:39]: oh, right, missed that.

**Review by apasel422 — APPROVED** [review]

**Pull request merged**